.gitignore

```diff
@@ -70,7 +70,7 @@ fastlane/test_output
 Packages/*
 *.xcodeproj/*
 Package.resolved
-.env
+.pf-env
```

Comment:
apparently newer versions of Docker use the `.env` file in the root for environment stuff so we can no longer use it as the place for our environment stuff. I looked for ways to get docker to ignore it, like the `.dockerignore` file but nothing worked.